cli/internal/config/config_file_test.go

```diff
@@ -9,6 +9,51 @@ import (
 	"github.com/vercel/turborepo/cli/internal/fs"
 )
 
+func Test_UserConfigPath(t *testing.T) {
+	fsys := afero.NewMemMapFs()
+
+	tests := []struct {
+		name string
+		fsys afero.Fs
+	}{
+		{
+			name: "get doesn't create a path",
+			fsys: fsys,
+		},
+	}
+	for _, tt := range tests {
```

Comment:
Are you planning to add more test cases to this? Otherwise, the table test pattern is probably overkill.

Reply:
Auto-generated test, 😅 I'll pull the loop.